pom.xml

```diff
@@ -108,6 +108,12 @@
       <classifier>jdk15</classifier>
       <scope>test</scope>
     </dependency>
+    
+    <dependency>
+	    <groupId>com.google.code.findbugs</groupId>
+	    <artifactId>findbugs</artifactId>
+	    <version>3.0.1</version>
+	</dependency>
```

Comment:
It does catch some things. I wish we could just configure it to not be annoying about trivial issues.